common/src/main/res/values/styles.xml

```diff
@@ -178,18 +178,38 @@
         <item name="android:textSize">14sp</item>
     </style>
 
+    <style name="MontserratRegular.red">
+        <item name="android:textColor">@color/red_300</item>
+    </style>
+
+    <style name="MontserratRegular.red.10">
```

Comment:
We should use styles based on our [design system](https://www.figma.com/file/azdJACb5WmivxYVhB5q46F/Android-%7C-Design-system?node-id=479%3A147) instead. This should be `Overline.Red`